content/analyst/competitive-tracker/competitive-tracker-reference-guide.md

```diff
@@ -0,0 +1,177 @@
+---
+lastUpdated: "02/23/2023"
+title: Competitive Tracker Reference Guide
+description: This article covers basic platform knowledge, reporting recommendations, and common use cases within the Competitive Tracker platform
+---
+
+## Basic Platform Knowledge
+
+
+* **[What is a user panel?](/analyst/general/what-is-a-user-panel)** Learn more about our panel and how it makes our data unique.
+* **[Common Terminology & Definitions.](/analyst/general/common-terminology-and-definitions)** This article covers our most common terms, phrases and words and what they actually mean within the platform.
+* **[Help! My searches have little-to-no email data.](/analyst/competitive-tracker/help-my-searches-have-little-to-no-data)** How you can contact our research team to troubleshoot little or missing data.
+
+---
+
+## Report Building
+
+* **[How to Export Data.](/analyst/inbox-and-design-tracker/how-to-export-your-eds-data)** Learn what page(s) contain exportable data for you to download and use outside of the platform.
```

Comment:
`/analyst/inbox-and-design-tracker/how-to-export-your-eds-data` -> `/analyst/inbox-and-design-tracker/how-to-export-data`